provider/datagen/src/bin/datagen/datagen.rs

```diff
@@ -0,0 +1,202 @@
+// This file is part of ICU4X. For terms of use, please see the file
```

Comment:
This file is moved, but unfortunately since I crated a new file with the old name, it shows as added/deleted